functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/BulkJobActionsSpec.groovy

```diff
@@ -0,0 +1,215 @@
+
+        then:
+        jobRespondsTrueTo(jobId1.toString()) { it.scheduleEnabled }
+    }
+
+    def "bulk disable schedule action disables the schedule"() {
+        given:
+        jobId1 = generateJob(["schedule-enabled": "true", "schedule-crontab": CRON_SCHEDULE_THAT_WONT_EXECUTE], "api-test-executions-running-scheduled.xml")
+        jobId2 = generateJob(["schedule-enabled": "true", "schedule-crontab": CRON_SCHEDULE_THAT_WONT_EXECUTE], "api-test-executions-running-scheduled.xml")
+
+        JobListPage jobsListPage = page JobListPage
+        jobsListPage.loadPathToNextUI SELENIUM_BASIC_PROJECT
+        jobsListPage.go()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.jobsActionsButton).click()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkEditButton).click()
+
+        when:
+        jobsListPage.waitForNumberOfElementsToBeMoreThan(jobsListPage.bulkJobRowItemsBy, 1)
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkSelectAllButton).click()
+
+        then:
+        def bulkPerformActionButton = jobsListPage.getBulkPerformActionButtonOnceClickable()
+        jobsListPage.waitIgnoringForElementToBeClickable(bulkPerformActionButton).click()
+
+        when:
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkDisableSchedulesAction).click()
+        jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.bulkConfirmActionYesButton).click()
+
+        then:
+        jobRespondsTrueTo(jobId1.toString()) { !it.scheduleEnabled }
+    }
+
+    def "job can be executed from the Job List Page modal"() {
+        given:
+        String jobToExecute = generateJob(["opt1-required": "false", "opt2-required": "false"])
+
+        JobListPage jobsListPage = page JobListPage
+
+        // TODO: Determine why the run button fails to open a modal in NextUI
+        //        jobsListPage.loadPathToNextUI SELENIUM_BASIC_PROJECT
+
+        jobsListPage.loadJobListForProject SELENIUM_BASIC_PROJECT
+        jobsListPage.go()
+
+        when:
+        def job1RunButton = jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.getExecuteJobInModalButton(jobToExecute))
+        job1RunButton.click()
+
+        def runJobNowButton = jobsListPage.waitIgnoringForElementToBeClickable(jobsListPage.getExecuteJobModalRunJobNowButton())
+        runJobNowButton.click()
+        jobsListPage.waitForUrlToContain('/execution/show/')
+
+        then:
+        ExecutionShowPage executionShowPage = page ExecutionShowPage
+        executionShowPage.validatePage()
+    }
+
+    /**
+     * Should be in JobUtils, but it's going through a refactor, thus it's here for now
+     * @param jobId
+     * @param c
+     * @param approximateWaitInSeconds
+     * @return
+     */
+    private boolean jobRespondsTrueTo(String jobId, Closure<Boolean> c, int approximateWaitInSeconds = 5) {
```

Comment:
GIven that many jobs would need to be checked, maybe a better option is to call the BaseContainer.hold() method and then check for the jobs instead of a waiting method per job.

Reply:
A compromise between hogging the test thread and doing checks in parallel.  I'd go for the 2nd. Please, see the changes in the test code.